src/applications/disability-benefits/all-claims/pages/addDisabilities.js

```diff
@@ -44,43 +35,31 @@ export const uiSchema = {
       ),
       'view:descriptionInfo': {
         'ui:description': () => (
-          <AlertBox isVisible status="info">
-            <p>
-              Below are some details that may be helpful to include when
-              describing your disability:
-            </p>
-            <ul>
-              <li>The part of your body that's affected</li>
-              <li>
-                If your disability is on the right side or left side of your
-                body
-              </li>
-              <li>The part of your body that isn't working right</li>
-            </ul>
-          </AlertBox>
+          <div>
```

Comment:
Don't we usually put this kind of thing in the content directory?

Reply:
Pfffffft. Yeah. You're right. Good call.